Makefile

```diff
@@ -15,4 +15,8 @@ lint-fix:
 lint:
 	./make/tasks/lint.sh
 lint/lockfiles:
-	./make/tasks/lint/lockfiles.sh
\ No newline at end of file
+	./make/tasks/lint/lockfiles.sh
+release:
+	./make/tasks/release/bundle.sh $(ARGS)
+release/bundle:
+	./make/tasks/release/bundle.sh $(ARGS)
```

Comment:
Why 2 commands with same script?

Reply:
Once the work is complete, the former will run additional steps. It's not implemented yet.